TESTS/assumptions/SPI/SPI.cpp

```diff
@@ -1,3 +1,8 @@
+// check if SPI is supported on this device
+#if !DEVICE_SPI
+    #error SPI is not supported on this platform, add 'DEVICE_SPI' definition to your platform.
```

Comment:
Oh nope, I was just copy pasting from the TESTS/api/SPI test and apparently that one gets it wrong as well! I will send a patch! Thanks for the catch!